region/norfair/east/Rising Tide.json

```diff
@@ -100,6 +100,11 @@
         {"refill": ["Energy", "Missile", "Super"]}
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": ["h_canHeatedCrystalFlash"]
```

Comment:
This breaks the roll under strat 🤔

Reply:
Ok, added entrance conditions to the "Roll Under" strats, which prevents a CF strat from happening in the middle. I used 
a comeInRunning with 0 tiles. Is that accurate, or do these actually need some speed from the previous room?

Reply:
It was only a problem for this strat, because the lava is rising.  `comeInRunning: 0` is a good way to write it, there is a different strat that would come in with speed for a mockball.

Reply:
Ok, `comeInRunning` will also require the other room to have normal physics (air or Gravity). So we may want to refine this at some point, e.g. by adding a new entrance condition that doesn't care about the neighboring room's physics, or add a property to the existing entrance conditions to override implicit physics requirements.